src/Sylius/Behat/Context/Api/Shop/LoginContext.php

```diff
@@ -27,6 +27,14 @@ public function __construct(ApiSecurityClientInterface $client)
         $this->client = $client;
     }
 
+    /**
+     * @Given there is the visitor
+     */
+    public function iAmAVisitor(): void
```

Comment:
Is it used somewhere?

Reply:
https://github.com/Sylius/Sylius/pull/11806/files#diff-6e397cfc7a6739ac80d9d7bc67120c01R191